src/services/transcript.ts

```diff
@@ -70,8 +72,14 @@ const categorizeItems = async (memberships: MembershipHistory[], jobs: StudentEm
 
   groupedMembershipHistory.activities = memberships;
 
-  groupedMembershipHistory.experiences.sort(
-    (a, b) => getExperienceEndDate(b) - getExperienceEndDate(a),
+  // sorting job title by alphabetical order
+  groupedMembershipHistory.experiences.sort((a, b) => getJobTitle(a).localeCompare(getJobTitle(b)));
```

Comment:
```suggestion
```
this is made redundant by the sort below

Reply:
the order of job titles is not alphabetical when the first sorting code is deleted. 

This is a transcript with sorting alphabetically
![train_after](https://github.com/gordon-cs/gordon-360-ui/assets/149107617/0d815fe5-9793-4672-bec5-2ca372429514)

This is a transcript after deleting the first sorting codes. The gray lines are for css test, ignore them.
![Screenshot from 2024-06-12 08-45-09](https://github.com/gordon-cs/gordon-360-ui/assets/149107617/57e5852a-b052-4eaa-b932-6e7a793927be)